modules/mobile/spec/request/addresses_request_spec.rb

```diff
@@ -10,8 +10,14 @@
   before { iam_sign_in(user) }
 
   let(:user) { FactoryBot.build(:iam_user) }
-  let(:address) { build(:va_profile_address, vet360_id: user.vet360_id) }
   let(:json_body_headers) { { 'Content-Type' => 'application/json', 'Accept' => 'application/json' } }
+  let(:address) do
```

Comment:
This is a bit nitpicky, but I'd suggest making this a separate test, like "it nullifies province if address is not domestic". One reason I say that is that some of the tests refer to `with a valid address` and this is not a valid address. If you called `address.valid?` on this it would be false.

Reply:
I was thinking to make sure it didn't cause any issues with any other endpoints. Since that private method  `address_params` is used in every endpoint, figured it'd be better to have it run the condition on every spec rather then make a whole new spec for each endpoint to test it.